modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectEntryValuesException.java

```diff
@@ -197,14 +199,13 @@ public String getObjectFieldName() {
 		extends ObjectEntryValuesException {
 
 		public InvalidObjectStateTransition(
-			ObjectState sourceObjectState, ObjectState targetObjectState) {
+			ObjectState sourceObjectState, ObjectState targetObjectState,
+			Serializable sourceObjectName, Serializable targetObjectName) {
 
 			super(
 				String.format(
-					"Object state ID %d cannot be transitioned to object " +
-						"state ID %d",
```

Comment:
@kiwm,

we need to keed this message in the tomcat log, but in the UI we need to display the name.

I believe you can apply a similar approach used in [these changes](https://github.com/liferay-objects/liferay-portal/pull/1675/commits/f14e0b9a82eb9cb6b5e4a694c3ee1e76758b9143#diff-ca8e28cfa773ca0d94b378df9c21b7cd5f3000ce7d694397194b469811f6e607R46-R47), it basically store two messages one that is logged and another that will be displayed for customers.

feel free to contact me if you have questions.

Best Regards,
Feliphe